src/utils/direct-messages.ts

```diff
@@ -26,7 +30,18 @@ import DMRoomMap from "./DMRoomMap";
 import { isJoinedOrNearlyJoined } from "./membership";
 import dis from "../dispatcher/dispatcher";
 import { privateShouldBeEncrypted } from "./rooms";
+import { LocalRoom, LocalRoomState, LOCAL_ROOM_ID_PREFIX } from "../models/LocalRoom";
+import * as thisModule from "./direct-messages";
```

Comment:
Is this to help with testing? Seems a bit fishy..

Reply:
Yes. If you don't import it from the same module it will call the function locally and you cannot mock parts of it.
Another solution would be splitting it up into separate modules (files).

What do you think?

Reply:
@kerryarchibald I split the file up